supabase/migrations/20250922191246_initial_tables.sql

```diff
@@ -0,0 +1,16 @@
+create table assets_unlocked_events (
+  unlock_sequence character varying not null,
+  recipient character varying not null,
+  token character varying not null,
+  sender character varying not null,
+  amount character varying not null,
+  chain int not null,
+  block_time numeric not null,
+  constraint assets_unlocked_events_pkey primary key (unlock_sequence)
+);
+
+create table signatures (
```

Comment:
Probably:
```suggestion
create table attestations_signatures (
```
as in the Go code we have:
```
	AttestationSignaturesTable = "attestations_signatures"
```